INSTALLATION.md

```diff
@@ -189,7 +189,7 @@ This guide provides step by step instructions to deploy PouchContainer on bare m
 As a developer, you may need to build and test PouchContainer binaries via source code. To build pouchd which is so-called "PouchContainer Daemon" and pouch which is so-called "PouchContainer CLI", the following build system dependencies are required:
 
 * Linux Kernel 3.10+
```

Comment:
I think we need to update this part, this part is never accurate.
CentOS, 3.10-327
Ubuntu, 4.0?

Could someone could help me to make it clear? @alibaba/pouch

Reply:
I will do that. @allencloud